db/migrate/20230213110327_create_entities.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEntities < ActiveRecord::Migration[7.0]
+  def change
+    create_table :entities do |t|
+      t.string :name
+      t.integer :amount
+      t.references :user, null: false, foreign_key: true
+
```

Comment:
- [ ] Great job working on this capstone. It looks awesome. There is an issue we need to look at. From the Entity-Relationship diagram, we have a field of AuthorID which is absent in your entity migration file. If you can recall from the BlogApp project you did in prelude to this capstone, there was a similar E-R diagram in which Author table was not on the E-R diagram but the field of AuthorID was an alias from the User table. Kindly take re-look at this issue